src/tests/routes/test_accesses.py

```diff
@@ -7,49 +7,72 @@
 
 from app import db
 from app.api import crud
-from tests.conf_test_db import populate_db
+from app.api.security import create_unlimited_access_token
+from tests.db_utils import populate_db
+
 
 ACCESS_TABLE = [
     {"id": 1, "login": "first_login", "hashed_password": "hashed_pwd", "scopes": "me"},
-    {"id": 2, "login": "second_login", "hashed_password": "hashed_pwd", "scopes": "me"},
+    {"id": 2, "login": "second_login", "hashed_password": "hashed_pwd", "scopes": "admin"},
+    {"id": 3, "login": "third_login", "hashed_password": "hashed_pwd", "scopes": "device"},
 ]
 
-
-@pytest.mark.asyncio
-async def test_get_access(test_app_asyncio, test_db, monkeypatch):
-    # Sterilize DB interactions
+@pytest.fixture(scope="function")
+async def init_test_db(monkeypatch, test_db):
     monkeypatch.setattr(crud.base, "database", test_db)
     await populate_db(test_db, db.accesses, ACCESS_TABLE)
 
-    response = await test_app_asyncio.get("/accesses/1")
-    assert response.status_code == 200
-    assert response.json() == {k: v for k, v in ACCESS_TABLE[0].items() if k != "hashed_password"}
-
 
 @pytest.mark.parametrize(
-    "access_id, status_code, status_details",
+    "access_idx, access_id, status_code, status_details",
```

Comment:
why idx and not ids?

Reply:
simply because it was more cumbersome to retrieve the correct ID from ACCESS_TABLE :sweat_smile: